src/test/cls_2pc_queue/test_cls_2pc_queue.cc

```diff
@@ -544,7 +544,7 @@ TEST_F(TestCls2PCQueue, Cleanup)
   aborter.join();
 
   ASSERT_GT(cleaned_reservations, 0);
-  ASSERT_GT(committed_reservations, 0);
+  ASSERT_EQ(committed_reservations + cleaned_reservations, number_of_ops*max_workers);
```

Comment:
due to different timing, it is not known how man of the reservations would be considered as stale and how many not.
so, assertion is on the fact that the overall numbers must match